From d5f87ddb46a2b352aa6a2b0d59f2ecc823df6f64 Mon Sep 17 00:00:00 2001
From: Susheel Nyamala <snyamala@codeaurora.org>
Date: Wed, 6 Mar 2019 20:29:38 +0530
Subject: [PATCH 4/9] Add support for retry with new DDS API and update
 QtiPhoneSwitcher

Modify access specifier of PhoneSwitcher members to support retry with new DDS API,
and QtiPhoneSwitcher update

Change-Id: Ibc6960a110d6e5927b76de35b76b9084da0161ae
CRs-Fixed: 2406274
---
 src/java/com/android/internal/telephony/PhoneSwitcher.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/com/android/internal/telephony/PhoneSwitcher.java b/src/java/com/android/internal/telephony/PhoneSwitcher.java
index 6a34a6278..151c10eef 100644
--- a/src/java/com/android/internal/telephony/PhoneSwitcher.java
+++ b/src/java/com/android/internal/telephony/PhoneSwitcher.java
@@ -242,7 +242,7 @@ public class PhoneSwitcher extends Handler {
     private static final int EVENT_EMERGENCY_TOGGLE               = 105;
     private static final int EVENT_RADIO_CAPABILITY_CHANGED       = 106;
     private static final int EVENT_OPPT_DATA_SUB_CHANGED          = 107;
-    private static final int EVENT_RADIO_AVAILABLE                = 108;
+    protected static final int EVENT_RADIO_AVAILABLE                = 108;
     // A call has either started or ended. If an emergency ended and DDS is overridden using
     // mEmergencyOverride, start the countdown to remove the override using the message
     // EVENT_REMOVE_DDS_EMERGENCY_OVERRIDE. The only exception to this is if the device moves to
-- 
2.28.0

